elasticsearch_collectd.py

```diff
@@ -537,15 +824,41 @@ def configure_callback(conf):
         elif node.key == 'Indexes':
             ES_INDEX = node.values
         elif node.key == 'EnableIndexStats':
-            ENABLE_INDEX_STATS = bool(node.values[0])
+            ENABLE_INDEX_STATS = _bool(node.values[0])
         elif node.key == 'EnableClusterHealth':
-            ENABLE_CLUSTER_STATS = bool(node.values[0])
+            ENABLE_CLUSTER_STATS = _bool(node.values[0])
         elif node.key == 'Interval':
             COLLECTION_INTERVAL = int(node.values[0])
+        elif node.key == 'IndexInterval':
+            INDEX_INTERVAL = int(node.values[0])
+        elif node.key == "DetailedMetrics":
+            DETAILED_METRICS = _bool(node.values[0])
+        elif node.key == "ThreadPools":
+            for i in node.values:
+                CONFIGURED_THREAD_POOLS.add(i)
+            # Include required thread pools (search and index)
+            CONFIGURED_THREAD_POOLS.add('search')
```

Comment:
this means that those defaults won't be set if the config doesn't contain ThreadPools setting